data/poprox-ingest.sql

```diff
@@ -5,49 +5,49 @@ INSERT INTO newsletters
 SELECT DISTINCT
     newsletter_id, account_id, created_at,
     recommender_name, recommender_version, recommender_hash
-FROM 'POPROX/newsletters.parquet';
+FROM 'data/POPROX/newsletters.parquet';
 
 INSERT INTO newsletter_articles
 SELECT
     newsletter_id, article_id, position, headline, subhead
-FROM 'POPROX/newsletters.parquet';
+FROM 'data/POPROX/newsletters.parquet';
 CREATE INDEX newsletter_article_idx ON newsletter_articles (article_id);
 
 INSERT INTO clicks
 SELECT account_id, newsletter_id, article_id, clicked_at
-FROM 'POPROX/clicks.parquet'
+FROM 'data/POPROX/clicks.parquet'
 ORDER BY account_id, newsletter_id;
 CREATE INDEX click_account_idx ON clicks (account_id);
 CREATE INDEX click_newsletter_idx ON clicks (newsletter_id);
 
 INSERT INTO interests
 SELECT account_id, entity_id, entity_name, preference
-FROM 'POPROX/interests.parquet'
+FROM 'data/POPROX/interests.parquet'
 ORDER BY account_id;
 
 INSERT INTO candidate_articles
 SELECT
     article_id, headline, subhead, url,
     linked_articles, raw_data, published_at, created_at, body
-FROM 'POPROX/articles.parquet'
+FROM 'data/POPROX/articles.parquet'
 ORDER BY created_at;
 
 INSERT INTO candidate_article_mentions
 SELECT mention_id, article_id, source, relevance, entity
-FROM 'POPROX/article_mentions.parquet'
+FROM 'data/POPROX/article_mentions.parquet'
 ORDER BY article_id;
 CREATE INDEX candidate_article_mention_article_idx ON candidate_article_mentions (article_id);
 
 INSERT INTO clicked_articles
 SELECT
     article_id, headline, subhead, url,
     linked_articles, raw_data, published_at, created_at
-FROM 'POPROX/clicked/articles.parquet'
+FROM 'data/POPROX/clicked/articles.parquet'
 ORDER BY article_id;
 
 INSERT INTO clicked_article_mentions
 SELECT mention_id, article_id, source, relevance, entity
-FROM 'POPROX/clicked/article_mentions.parquet'
+FROM 'data/POPROX/clicked/article_mentions.parquet'
```

Comment:
What command were you running that caused a failure due to these paths? This script is intended to be run from within the `data` directory, and is run in that manner in the DVC automation, so the original paths should be correct.